workflow-templates/phpunit-mysql.yml

```diff
@@ -34,7 +34,7 @@ concurrency:
 
 jobs:
   phpunit-mysql:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
Just a question, the images are now "ubuntu-latest"?
Can apps pin it to specific ubuntu versions (e.g. to support old PHP versions on older stable branches)?

Reply:
Could we then just tag it as such, like:
```
runs-on: self-hosted:22.04
```

because if we later upgrade and use ubuntu 24.04 lts e.g. we would need to modify the github action on all stable branches, instead of "hey there is a new image, let's use it for new branches"

Reply:
We could specify explicit tags yes.
Let's put this PR on hold, we're having an internal discussion about the next steps with Marcel, i'll communicate more clearly about this later, sorry for the ping, I should have thought this more